frontend/src/components/Workspace/FlowChart/TreeView.tsx

```diff
@@ -170,6 +175,10 @@ const InputNodeComponent = memo(function InputNodeComponent({
           reactFlowNodeType = REACT_FLOW_NODE_TYPE_KEY.BehaviorFileNode
           fileType = FILE_TYPE_SET.BEHAVIOR
           break
+        case FILE_TYPE_SET.MATLAB:
+          reactFlowNodeType = REACT_FLOW_NODE_TYPE_KEY.MatlabFileNode
+          // fileType = FILE_TYPE_SET.MATLAB
```

Comment:
Is this expected comment out?

Reply:
@ReiHashimoto i fixed it, pls check it again